docs/tutorial.md

```diff
@@ -452,20 +475,23 @@ A definition for a service is the namespace that contains all the operations for
 
 Here's an example that uses these to define a Pet Store service:
 
-```
+```cadl
 @serviceTitle("Pet Store Service")
 @serviceVersion("2021-03-25")
-@produces("application/json", "image/png")
-@consumes("application/json")
+@Cadl.Rest.produces("application/json", "image/png")
+@Cadl.Rest.consumes("application/json")
```

Comment:
This was annoying and confusing. You can't have `using Cadl.Rest`; and `using Cadl.Http` at the same time even if you don't actually use conflicting definitions between the namespaces. The error you get is that there's a duplicate `delete` defined in both `rest.cadl` and `http.cadl`.  It would be better if you only got an error if you tried to use `delete` and it squiggled your ambiguous `delete`. Right now, you don't even get the right feedback in the IDE because it's "blaming" the libraries.

I hadn't realized this subtle (and unfortunate, IMHO) difference from C# until I went through this exercise. I will file something to make it more like C#. But this is likely to be a lot of discussion and work and push into v2 or beyond, and still not solve every case, so we should still work to avoid any conflicts in our commonly co-used namespaces.

Reply:
Filed #79

Reply:
Yep, this is the issue I've been talking about a few different times in meetings.  My next PR will actually rename the conflicting decorators in `Cadl.Rest` so that they don't conflict any longer, that way you can use it with `Cadl.Http` with no trouble.

Reply:
I see. I hadn't appreciated how bad it was. I thought it would merely be as annoying as in C# when there's a conflict, but current Cadl rules are much more annoying than that. That said, we avoid conflicts in C# and people complain about the less annoying rules there too. So renaming is still good even if we do #79. 😁